gbmi/utils/__init__.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import annotations
+
+import copy
+import itertools
+import os
+import subprocess
+import sys
+
+from pathlib import Path
+from typing import TypeVar, List, Dict
+from transformer_lens import HookedTransformer
+
+import numpy as np
+import torch
+from jaxtyping import Float
+from lightning import Callback
+from numpy.random import Generator
+from torch import Tensor
+
+PROJECT_ROOT = Path(__file__).parent.parent.parent
```

Comment:
This seems fragile, especially since it goes outside the top-level package, and with respect to refactoring.  Maybe we want to have a `get_git_project_root`?  Or ensure it's a subdirectory of the package?